versions/development/SPEC.md

```diff
@@ -538,22 +526,24 @@ Below are the valid results for operators on types.  Any combination not in the
 |`Int`|`>=`|`Int`|`Boolean`||
 |`Int`|`<`|`Int`|`Boolean`||
 |`Int`|`<=`|`Int`|`Boolean`||
-|`Int`|`+`|`String`|`String`||
-|`String`|`+`|`Float`|`String`||
-|`String`|`+`|`Int`|`String`||
-|`String`|`+`|`String`|`String`||
+|`String`|`+`|`String`|`String`|Concatenation|
 |`String`|`==`|`String`|`Boolean`||
 |`String`|`!=`|`String`|`Boolean`||
 |`String`|`>`|`String`|`Boolean`||
 |`String`|`>=`|`String`|`Boolean`||
 |`String`|`<`|`String`|`Boolean`||
 |`String`|`<=`|`String`|`Boolean`||
+|`File`|`==`|`File`|`Boolean`||
+|`File`|`!=`|`File`|`Boolean`||
```

Comment:
The table is alphabetical, so these should be higher up, where they originally were.